### ISSUE TITLE
knit error for awesome cv template

### ISSUE THREAD

**Issue by statKim** [mitchelloharawild/vitae#154]
I would like to ask for advice on this error when I knit the default "awesome-cv" template in vitae package. (I saw the same issue for this problems.)

Error in gsub("<<PANDOC_PATH>>", rmarkdown::find_pandoc()$dir, fixed = TRUE,  : 
  input string 2 is invalid in this locale
Calls: <Anonymous> ... do.call -> <Anonymous> -> cv_document -> cat -> gsub
Execution halted

And I found the source code where this problem occurs. (<https://github.com/mitchelloharawild/vitae/blob/master/R/cv_document.R#L25>)
```r
> gsub("<<PANDOC_PATH>>", rmarkdown::find_pandoc()$dir, fixed = TRUE,
+      readLines(system.file("multiple-bibliographies.lua", package = "vitae", mustWork = TRUE)))
Error in gsub("<<PANDOC_PATH>>", rmarkdown::find_pandoc()$dir, fixed = TRUE,  : 
  input string 2 is invalid in this locale
```

However, if I add the option `encoding = "UTF-8"` for `readLines()`, it does not occur the errors.
```r
gsub("<<PANDOC_PATH>>", rmarkdown::find_pandoc()$dir, fixed = TRUE,
+      readLines(system.file("multiple-bibliographies.lua", package = "vitae", mustWork = TRUE), encoding = "UTF-8"))
```

This is my session info about R.
```r
> sessionInfo()
R version 4.0.3 (2020-10-10)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19041)

Matrix products: default

locale:
[1] LC_COLLATE=Korean_Korea.949  LC_CTYPE=Korean_Korea.949    LC_MONETARY=Korean_Korea.949 LC_NUMERIC=C                
[5] LC_TIME=Korean_Korea.949    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
 [1] assertthat_0.2.1  crayon_1.4.0      digest_0.6.27     dplyr_1.0.4       R6_2.5.0          DBI_1.1.1        
 [7] lifecycle_0.2.0   magrittr_2.0.1    evaluate_0.14     pillar_1.4.7      rlang_0.4.10      ellipsis_0.3.1   
[13] vctrs_0.3.6       generics_0.1.0    rmarkdown_2.6     tools_4.0.3       glue_1.4.2        purrr_0.3.4      
[19] xfun_0.20         yaml_2.2.1        compiler_4.0.3    pkgconfig_2.0.3   htmltools_0.5.1.1 vitae_0.4.1      
[25] tidyselect_1.1.0  knitr_1.31        tibble_3.0.6     
```

I think it can be solved if I set the system locale to "UTF-8", but I couldn't find the solution in windows 10.

Could you give me some advice to fix this problems?

Thank you for reading it!

**Comment by mitchelloharawild** [comment]
Setting the encoding of the file sounds like a good solution. Would you like to make a pull request with this change?

**Comment by statKim** [comment]
I made a pull request about this change. Thank you.

**Comment by mitchelloharawild** [comment]
Resolved by #155, thanks!
